mock/backend.go

```diff
@@ -149,6 +149,18 @@ func (mr *MockBackendMockRecorder) RetryImportDelay() *gomock.Call {
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "RetryImportDelay", reflect.TypeOf((*MockBackend)(nil).RetryImportDelay))
 }
 
+// ShouldPostProcess mocks base method
+func (m *MockBackend) ShouldPostProcess() bool {
+	ret := m.ctrl.Call(m, "ShouldPostProcess")
```

Comment:
I don't understand the purpose of this function, is it calling itself?

Reply:
This file is generated by [mockgen](https://github.com/golang/mock), no need to review its content 😅